src/components/pages/lessons/overlay/next-up-overlay.tsx

```diff
@@ -11,7 +11,7 @@ const NextUpOverlay: React.FunctionComponent<{
 }> = ({lesson, nextLesson, onClickRewatch = noop}) => {
   const courseImage = lesson?.collection?.square_cover_480_url
   return (
-    <>
+    <div className="bg-gray-800 bg-opacity-90 grid place-items-center w-full h-full absolute z-10 top-0 left-0">
```

Comment:
let this exist as an overaly on top of the player

@DrShpongle this made it look kind of bad.